rpcserver.go

```diff
@@ -761,6 +761,22 @@ func (r *rpcServer) OpenChannel(in *lnrpc.OpenChannelRequest,
 			"non-negative number")
 	}
 
+	var minConfs int32
```

Comment:
Doesn't look like it is ever set in the case `in.MinConfs > 0`

Reply:
Fixed!